src/track/trackmetadatataglib.cpp

```diff
@@ -1691,6 +1916,15 @@ void importTrackMetadataFromMP4Tag(TrackMetadata* pTrackMetadata, const TagLib::
         pTrackMetadata->refTrackInfo().setGrouping(grouping);
     }
 
+    QString work;
+    if (readMP4Atom(tag, "\251wrk", &work)) {
```

Comment:
I have been stumbled over \251. Is that a kind of legacy? Do you know the origin?  From the latin-1 code page it is the ©
I think we should change it to the \xa9 notation everywhere. I have found it in this notation in other references and for me the octal decoding is kind of untypical unfamiliar. Github finds 15 vs 126 for mvn